<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="bigPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="thing">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>529</y>
      <width>801</width>
      <height>171</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="DeviceParams">
     <attribute name="title">
      <string>Device</string>
     </attribute>
     <widget class="QLineEdit" name="J">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>J </string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="Kpp">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>51</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>K</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gammaK">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lm0">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Lambda</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kerr K</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="R1">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>9</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="Lc">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>49</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>50</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Length</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>R1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="R2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>9</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="df">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>49</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>50</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Detunning</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>R2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="QclParams">
     <attribute name="title">
      <string>QCL</string>
     </attribute>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mu</string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Lmod</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>11</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>T1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="T1">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>T2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Lmod">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>49</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="T2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="n">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>9</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>n</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mu">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Waveguide</string>
     </attribute>
     <widget class="QLineEdit" name="Gamma">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Aw</string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="aw">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Gamma</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Simulation</string>
     </attribute>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>59</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>gcfac</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="h">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>19</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>19</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>dt/Tr</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>59</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ls</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dtperTr">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>19</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>19</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Courant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numTr">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>18</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="gcfac">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>59</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>19</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>#Tr</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Crnt">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>18</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>h</string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>99</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>#Save</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="maxsave">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>99</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Phi0</string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="Ls">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string comment="0">Lorentzian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="1">Parabolic</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="initphi">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Fundamental</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Harmonic N</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cosine N</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Random</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="gc">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Gain Curvature</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="useN2N3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>50</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Use N2N3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Plot</string>
     </attribute>
     <widget class="QCheckBox" name="plotprogress">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot Progress</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="plottheory">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot Theory</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="shifttomin">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Scale Phase</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Simulate">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>500</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">QTimer</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
